project/src/components/svg-namespace/svg-namespace.tsx

```diff
@@ -0,0 +1,44 @@
+function SvgNamespace(): JSX.Element {
+
+  return (
```

Comment:
```suggestion
function SvgNamespace(): JSX.Element {
  return (
```

SvgNamespace - мне не понятно по названию, что это за иконка.